src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -230,7 +230,7 @@ ul.dock  > li.prev img.app-logo {
 }
 
 #mobile-dock {
-  background-color: @decorator-alt-color;
+  background-color: @bg-on-bg-inverse-color;
```

Comment:
The name of this variable should be: `@mno-dock-mobile-bg-color`.
The dock variables must respect the convention @mno-dock-*.

`@bg-on-bg-inverse-color` is not explicit, please do not use this.